docs/src/main/asciidoc/logging.adoc

```diff
@@ -672,6 +666,82 @@ implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
 
 . Verify whether the logs generated by the added library adhere to the same format as the other Quarkus logs.
 
+=== Using MDC (Mapped Diagnostic Context)
+
+Quarkus overrides log MDC (Mapped Diagnostic Context) to improve the compatibility with its reactive core.
+
+==== Adding and Reading MDC data
```

Comment:
Do the verb-form rules apply to subheadings too? If so this should be "Add and read MDC data" I guess. Anyway "Reading" probably shouldn't be capitalized.

Reply:
I'm not sure if the rule applied to sub-sub-sub headings. I decided to ignore it at that level.